gazebo_plugins/CMakeLists.txt

```
@@ -448,6 +448,23 @@ if(ENABLE_PROFILER)
   target_link_libraries(gazebo_ros_projector ${ignition-common3_LIBRARIES})
 endif()
 
+# gazebo_ros_wheel_slip
+if(NOT GAZEBO_VERSION VERSION_LESS 9.5)
+  add_library(gazebo_ros_wheel_slip SHARED
+    src/gazebo_ros_wheel_slip.cpp
+  )
+  ament_target_dependencies(gazebo_ros_wheel_slip
+    "gazebo_dev"
+    "gazebo_ros"
+    "rclcpp"
+    "std_msgs"
+  )
+  target_link_libraries(gazebo_ros_wheel_slip
+    WheelSlipPlugin
+  )
+  ament_export_libraries(gazebo_ros_wheel_slip)
```

Looks like there is an additional change needed for the include directories
```suggestion
  target_include_directories(gazebo_ros_wheel_slip PUBLIC include)
  ament_export_libraries(gazebo_ros_wheel_slip)
```